sdks/python/apache_beam/runners/portability/fn_api_runner.py

```diff
@@ -987,21 +1039,26 @@ def wrapper(constructor):
     return wrapper
 
   @classmethod
-  def create(cls, environment, state, provision_info):
+  def create(cls, environment, state, provision_info, grpc_server=None):
```

Comment:
Any reason to let grpc_server be omitted? Might be better to be an error to not explicitly provide it.